### ISSUE TITLE
Sensor Node for OpenMQTTGateway?

### ISSUE THREAD

**Issue by knopserl** [1technophile/low_power_sensor#1]
This looks like something I wanted to build in conjunction with the OpenMQTTGateway.
I guess this sketch could be used to build a sensor node for OpenMQTTGateway right?
The RF module (RCSwitch?) can be replaced by a RFM95 lib right and can then fit to the OpenMQTTGateway with also the RFM95 library (once it's available)?

**Comment by 1technophile** [comment]
Yes we could do it like that. But I think it will impact a lot the code.

**Comment by knopserl** [comment]
thanks for the  reply, I know there are a lot of wishes :-) I would be patient. Thought as you might accept the RFM95 for OpenMQTTGateway, it would be nice to have a matching device/node like the low_power_sonsor which would fit nicely to the MQTT Gateway/server.
Or do you have a better idea how to build a RF low power sensor for the MQTTGateway? any other project/sketch which would fit better and create less effort for you?
